install_config/aggregate_logging.adoc

```diff
@@ -442,6 +438,17 @@ remove all logging persistent data including PVC, set
 `openshift_logging_install_logging` to 'false' to trigger uninstallation and
 `openshift_logging_purge_logging` to 'true'. The default is set to 'false'.
 
+|`openshift_logging_ops_log_source`
+|
+
+Permitted values are `file` and `journald`. If `file`, use `/var/log/messages` as the
+operations/system log source. If `journald`, use `journald`. The default should be
+`journald`. This will override settings applied to `USE_JOURNAL/use_journal`.
+
+If none of the previously listed parameters are set, expect the following behavior.
+System/operations logs are read from `journald`, and container/application logs
+should be read from the settings applied to Docker `--log-driver`.
+
```

Comment:
There is no `openshift_logging_ops_log_source`.  System logs are always read from the journal now.  For container logs, fluentd will figure out which log driver docker is using - json-file or journald - and will read the logs from that source, automatically.